### PR TITLE
Move issues assigned check to probot

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
Moved issues assigned check to probot. This feature has been tested at https://github.com/jameesjohn/comment-on-pr/issues/10#issuecomment-650818961

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by jameesjohn** [comment]
@ankita240796, doing this will make tests fail because one of the implementations will not be tested and hence, the overall coverage will be reduced.

File: lib/checkIssueAssignee.js [hunk 1]
```diff
@@ -16,74 +16,71 @@
...
   // Catch and log any error that will occur while accessing google sheets api.
   try {
-    core.info('Checking if ' + assignee.login + ' has signed the CLA');
+    console.log('Checking if ' + assignee.login + ' has signed the CLA');
     const assigneeHasSignedCla = await hasSignedCla(assignee.login);
     if (!assigneeHasSignedCla) {
-      core.info(assignee.login + ' has not signed the CLA');
+      console.log(assignee.login + ' has not signed the CLA');
 
       const commentBody = 'Hi @' + assignee.login + ', you need to sign the ' +
```
Comment (ankita240796) [inline lib/checkIssueAssignee.js:63]: Add another line in the comment: I am unassigning you for now, feel free to assign yourself once you have signed the CLA.

Comment (jameesjohn) [reply lib/checkIssueAssignee.js:63]: Done.

**Comment by jameesjohn** [comment]
Issue created at https://github.com/oppia/oppiabot/issues/98

**Pull request merged**